src/composables/queries/useTokenPricesQuery.ts

```diff
@@ -27,37 +27,20 @@ const PER_PAGE = 1000;
  */
 export default function useTokenPricesQuery(
   addresses: Ref<string[]> = ref([]),
-  pricesToInject: Ref<Record<string, number>> = ref({}),
+  pricesToInject: Ref<TokenPrices> = ref({}),
   options: UseQueryOptions<QueryResponse> = {}
 ) {
   const { networkId } = useNetwork();
   const queryKey = reactive(
     QUERY_KEYS.Tokens.Prices(networkId, addresses, pricesToInject)
   );
-  const { currency } = useUserSettings();
-
-  // TODO: kill this with fire as soon as Coingecko supports wstETH
-  function injectWstEth(prices: TokenPrices): TokenPrices {
-    const stEthAddress = configService.network.addresses.stETH;
-    const wstEthAddress = configService.network.addresses.wstETH;
-    if (prices[stEthAddress]) {
-      const stETHPrice = prices[stEthAddress][currency.value] || 0;
-      prices[wstEthAddress] = {
-        [currency.value]: TOKENS.Prices.ExchangeRates.wstETH.stETH * stETHPrice
-      };
-    }
-
-    return prices;
-  }
```

Comment:
wstETH is now supported in Coingecko! Yay!

https://api.coingecko.com/api/v3/simple/token_price/ethereum?contract_addresses=0x7f39C581F595B53c5cb19bD0b3f8dA6c935E2Ca0&vs_currencies=usd